.travis.yml

```diff
@@ -3,13 +3,11 @@ language: ruby
 bundler_args: --without development
 before_install: rm Gemfile.lock || true
 rvm:
-  - 1.8.7
```

Comment:
Right, sorry, looks like it was https://github.com/rspec/rspec-core/issues/1864 that was causing the failure, I pinned the rspec version to < 3.2.0...